### PR TITLE
Edits to log forwarding per email

### PR DESCRIPTION
**Pull request by mburke5678**
Per email and Slack: https://coreos.slack.com/archives/GGUR75P60/p1582896919255000

"I saw a mistake in the yaml file. Type and name should be switched. This is also wrong in the example in the docs. "

### CODE REVIEW

**Comment by mburke5678** [comment]
@sichvoge @jcantrill  PTAL
I flipped the `name` and `type` parameters in the log forwarding CR, as I think the customer, Dennis Hergaarden, was suggesting. 
I also saw in his email that he needed to disable log forwarding and delete the log forward CR in order to make edits in the CR file. I added the steps here too.

**Comment by mburke5678** [comment]
@sichvoge @jcantrill PTAL

File: modules/cluster-logging-log-forwarding-configure.adoc [hunk 1]
```diff
@@ -96,12 +96,72 @@ spec:
...
-
++
 [NOTE]
 ====
 If you set `disableDefaultForwarding: true` you must configure a pipeline and output for all three types of logs, application, infrastructure, and audit. If you do not specify a pipeline and output for a log type, those logs are not stored and will be lost.
 ==== 
 
+== Updating the Log Forwarding Custom Resource
+
+If you make changes to the log forwarding outputs or pipelines, you need to disable log forwarding, edit the CR, then enable log forwarding again.
```
Comment (jcantrill) [inline modules/cluster-logging-log-forwarding-configure.adoc]: This really should be unnecessary.  If you have to do this then there is a bug in the code.

**Review by anpingli — COMMENTED** [review]
https://secureforward.offcluster.com:9200  --> https://secureforward.offcluster.com: 24224.   (Most of time, that is a fluentd receiver. The default port is 24224)

**Review by anpingli — COMMENTED** [review]
The other part LGTM

File: modules/cluster-logging-log-forwarding-configure.adoc (hunk 1 above)
Comment (anpingli) [reply modules/cluster-logging-log-forwarding-configure.adoc]: @mburke5678  Yes, It is unnecessary.  The change can be apply without disbale log forwarding. if the forward didn't take effect,  we will filed a bug to trace it.

Comment (mburke5678) [reply modules/cluster-logging-log-forwarding-configure.adoc]: @anpingli Thanks. I think I misunderstood a comment you made in Slack. I will follow up with https://github.com/openshift/openshift-docs/pull/20675 to remove this section,

**Pull request merged**